src/main/java/org/opentripplanner/routing/algorithm/RoutingWorker.java

```diff
@@ -63,7 +63,7 @@
      * and would also need to board trips starting tomorrow. If a search window that lasts more than
      * a day is used, this will need to be increased.
      */
-    private static final int ADDITIONAL_SEARCH_DAYS_AFTER_TODAY = 1;
+    private static final int ADDITIONAL_SEARCH_DAYS_AFTER_TODAY = 2;
```

Comment:
I did some test searches within Oslo, and the variance between searches is much higher than the variance between 1 and 2 additional search days.

The search time increased about 5% for long searches (Oslo - Trondheim). Even increasing the additional search days to 7 only increased the search time of long searches by about 15%.

How we filter trips by running period is something we could tune more in the future. This structure allows us to select trips between two exact DateTimes, and these could be set per request (for example based on distance).

Reply:
Great, the numbers are in the expected range and it would be easy to make this configurable in the future. One thing you can consider is to put it in the `RoutingRequest`(Internal). Then it would be configurable, but not available on he APIs.